source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -0,0 +1,144 @@
+﻿$(document).ready(function () {
+    var viewmodel;
+
+    var Calification = function (data) {
+        var self = this;
+        this.evaluation = new Evaluation();
+        this.template = new Template();
+        if (data) {
+            this.fromJs(data);
+        }
+    }
+
+    Calification.prototype.fromJs = function (data) {
+        this.evaluation.fromJs(data.Evaluation);
+        this.template.fromJs(data.Template);
+    }
+
+    Calification.prototype.toJs = function () {
+        return {
+            Evaluation: this.evaluation.toJs(),
+            Template: this.template.toJs()
+        };
+    }
+    Calification.prototype.load = function () {
+        $.getJSON("/Evaluations/api/getEvaluation/", function (model) {
+            viewmodel.fromJs(model);
+        });
+    }
+
+    var Evaluation = function (data) {
+        this.id = ko.observable('');
+        this.userName = ko.observable('');
+        this.responsibleId = ko.observable('');
+        this.fullName = ko.observable('');
+        this.currentPosition = ko.observable('');
+        this.seniority = ko.observable('');
+        this.period = ko.observable('');
+        this.project = ko.observable('');
+        this.updateProject = function () {
+            var project = { Project: this.project() };
+            $.ajax("/Evaluations/api/UpdateProject/", {
```

Comment:
What happen if one of these AJAX calls (also [line 25](https://github.com/CommonJobs/CommonJobs/pull/384/files#diff-1b28b31e2207786095b5affe916c595bR25)) fails?